packages/core/src/textures/resources/BaseImageResource.ts

```diff
@@ -10,13 +12,17 @@ import { ALPHA_MODES } from '@pixi/constants';
  */
 export class BaseImageResource extends Resource
 {
+    source: ImageSource;
+    noSubImage: boolean;
```

Comment:
This should be protected

Reply:
oh, both are public actually

Reply:
ah ok cool. 

Looking at the jsdocs should `source` be readonly?

Reply:
I recommend people to use `resource.source` in several cases. Yes, it wont actually work in this version because `baseTextrure.resource` is `Resource` and not `BaseImageResource` . We might resolve it later. 

@bigtimebuddy should we make it public or protected?